src/components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbContent/BreadcrumbContent.tsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import React, { ForwardedRef, forwardRef, useCallback, useMemo } from "react";
+import VibeComponentProps from "../../../../types/VibeComponentProps";
+import { iconSubComponentProps } from "../../../Icon/Icon";
+import "./BreadcrumbContent.scss";
+
+const ENTER_KEY = "Enter";
+const SPACE_KEY = " ";
+
+interface BreadcrumbContentProps extends VibeComponentProps {
+  isClickable?: boolean;
+  link?: string;
+  onClick?: () => void;
+  text?: string;
+  icon?: string | React.FC<iconSubComponentProps> | null;
```

Comment:
Let's convert to IconType?

Reply:
I think we should fix IconType. Current implementation results in these errors
<img width="818" alt="image" src="https://user-images.githubusercontent.com/104433616/197386106-8a1b0dd3-6084-48a0-b45c-54cff41195be.png">
Maybe in this case it should be like this `keyof JSX.IntrinsicElements`?